container-toolkit/index.md

```diff
@@ -23,16 +23,19 @@ Container Device Interface <cdi-support.md>
 docker-specialized.md
 ```
 
-The NVIDIA Container Toolkit enables users to build and run GPU-accelerated containers.
-The toolkit includes a container runtime [library](https://github.com/NVIDIA/libnvidia-container)
-and utilities to automatically configure containers to leverage NVIDIA GPUs.
+The NVIDIA Container Toolkit is a collection of libraries and utilities enabling users to build and run GPU-accelerated containers.
 
-```{image} https://cloud.githubusercontent.com/assets/3028125/12213714/5b208976-b632-11e5-8406-38d379ec46aa.png
-:width: 400
-```
+The Toolkit currently includes:
+
+* The NVIDIA Container Runtime (`nvidia-container-runtime`)
+* The NVIDIA Container Toolkit CLI (`nvidia-ctk`)
+* The NVIDIA CDI Hooks (`nvidia-cdi-hook`)
+* The NVIDIA Container Runtime Hook (`nvidia-container-runtime-hook`)
+* The NVIDIA Container CLI (`nvidia-container-cli`)
+* The NVIDIA Container Library (`libnvidia-container1`)
```

Comment:
We can think about linking out to the github repos of those components.

Reply:
I'm not sure about links to the GitHub repo(s) at this stage as that adds additional questions. For example, the first four components listed here are developed from the `nvidia-container-toolkit` repo and the last two from `libnvidia-container`. We are moving towards just *needing* a single one and maybe we can add the reference then?

Reply:
I agree, it might be too much have the links here to each component repo at this point in the docs. If we wanted to include links to the repo here, it might make sense to do that in a table format. So we could include a "Repo link" column in the and visually show that some components are in the same repo.

In general, I'm pro adding links and cross-referencing things. No real restrictions on adding links to github repos or other docs sites (like the k8s.io site) to help explain concepts. As long as its adding value and not chaos. I guess the only note there is if we have docs/explanations in a repo that should live in the docs site, then we should add it to the docs site, instead of linking to the repo from the docs.

Reply:
Thanks for the exhaustive feedback to both of you! I will leave this as-is.

> No real restrictions on adding links to github repos or other docs sites (like the k8s.io site) to help explain concepts.

Thanks for confirming! :)